include/hip/hcc_detail/texture_functions.h

```diff
@@ -23,14 +23,24 @@ THE SOFTWARE.
 #ifndef HIP_INCLUDE_HIP_HCC_DETAIL_TEXTURE_FUNCTIONS_H
 #define HIP_INCLUDE_HIP_HCC_DETAIL_TEXTURE_FUNCTIONS_H
 
-#include <hc.hpp>
-#include <hc_short_vector.hpp>
+#include <hip/hcc_detail/hip_vector_types.h>
 #include <hip/hcc_detail/hip_texture_types.h>
 
+#define TYPEDEF_VECTOR_VALUE_TYPE(SCALAR_TYPE) \
+typedef SCALAR_TYPE SCALAR_TYPE##2_vector_value_type __attribute__((ext_vector_type(2))); \
```

Comment:
Please prefix the typedef name with `__hip_`

Reply:
I just prefixed all the vector_value_types with __hip_